cmd/deploy.go

```diff
@@ -72,9 +76,19 @@ func deploy(ctx *cmdutil.Ctx) error {
 			defer deployGroup.Done()
 			perform(ctx, path, op)
 		}(path, op)
+
+		switch(op) {
+		case file.Update:
+			updateCount++
+		case file.Skip:
+			skipCount++
+		case file.Remove:
+			removeCount++
+		}
 	}
 
 	deployGroup.Wait()
+	fmt.Printf("Updated: %d, Removed: %d, Skipped: %d", updateCount, removeCount, skipCount)
```

Comment:
This works, except that the final line of output gets shown afterwards, e.g.

```
[development] Skipped templates/customers/account.liquid (Checksum: 4fe35d7a307346edefa4b53d554ab06b)
[development] Skipped templates/article.liquid (Checksum: 20defecae451efdebc626d58cee1a87f)
Updated 0, Removed 0, skipped 25
[development] Skipped config/settings_data.json (Checksum: e8175e268a84d3861e3074569e0f986c)
```
Any idea why that is @tanema ?